content/concept-amendments.md

```diff
@@ -0,0 +1,164 @@
+# Amendments #
+
+The Amendments system provides a means of introducing new features to the decentralized Ripple consensus network without causing disruptions. The amendments system works by utilizing the core consensus process of the network to approve any changes by showing continuous support before those changes go into effect. An amendment normally requires **80% support for two weeks** before it can apply.
+
+When an Amendment has been enabled, it applies permanently to all ledger versions after the one that included it. You cannot disable an Amendment, unless you introduce a new Amendment to do so.
+
+
+## Background ##
+
+Any changes to transaction processing could cause servers to build a different ledger with the same set of transactions. If only a portion of validators have upgraded to a new version of the software, this could cause anything from minor inconveniences (a minority of servers spend more time and bandwidth fetching the actual consensus ledger because they cannot rely on their own) to serious problems: consensus might be unable to validate new ledger versions because servers are unable to reach a consensus who agree to the exact same result.
```

Comment:
Good point. Even after revision, it's a bit of a challenge, but I think it's better:
> If only a portion of validators have upgraded to a new version of the software, this could cause anything from minor inconveniences to full outages. In the minor case, a minority of servers spend more time and bandwidth fetching the actual consensus ledger because they cannot build it using the transaction processing rules they already know. In the worst case, consensus might be unable to validate new ledger versions because servers with different rules could not reach a consensus on the exact ledger to build.

I wanted to say "cannot" instead of "could not" in the last sentence, but that makes it into a general statement about consensus that's not true. (Servers with different rules _can_ reach a consensus in some situations, namely, the cases where they omit anything they disagree on.)